Discord Mafia Bot .Net/Commands/Commands.cs

```diff
@@ -39,6 +39,11 @@ public class MiscCommands : ModuleBase
             {
                 await Context.Message.AddReactionAsync(new Emoji("🍪"));
             }
+            [Command("apple"), Summary("Gives you an apple!")]
```

Comment:
Please remove this, I'll add it to the actual commit in the end (and if I don't remind me) so it's fixed in with Zachie's newest commit.